clustermesh-apiserver/clustermesh/testdata/ciliumendpoints.txtar

```diff
@@ -2,6 +2,9 @@
 
 hive/start
 
+# Add two namespaces
+k8s/add namespace-1.yaml namespace-2.yaml
```

Comment:
Please extend this test (and similarly the others) by removing one namespace at the end, and checking that the corresponding entries are removed; then, add it back, and assert that they get added back again.

Reply:
Good idea. Done!